packages/app-page-builder/src/editor/plugins/elements/carousel/PeCarousel.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useRef, useEffect } from "react";
+import styled from "@emotion/styled";
+import { createRenderer, useRenderer, Elements } from "@webiny/app-page-builder-elements";
+import { Element } from "@webiny/app-page-builder-elements/types";
+import { useRecoilValue } from "recoil";
+import { elementWithChildrenByIdSelector } from "~/editor/recoil/modules";
+import { register } from "swiper/element/bundle";
+
+register();
```

Comment:
Let's also have this in a `useEffect` call. No need to have this called immediately.